packages/cli/workspace-loader/src/utils/convertOpenApiWorkspaceToFernWorkspace.ts

```diff
@@ -33,29 +22,42 @@ export async function convertToFernWorkspace(
             dependencies: {}
         },
         workspaceName: openapiWorkspace.workspaceName,
-        definition: {
-            // these files doesn't live on disk, so there's no absolute filepath
-            absoluteFilepath: AbsoluteFilePath.of("/DUMMY_PATH"),
-            rootApiFile: {
-                contents: definition.rootApiFile,
-                rawContents: yaml.dump(definition.rootApiFile)
-            },
-            namedDefinitionFiles: {
-                ...mapValues(definition.definitionFiles, (definitionFile) => ({
-                    // these files doesn't live on disk, so there's no absolute filepath
-                    absoluteFilepath: AbsoluteFilePath.of("/DUMMY_PATH"),
-                    rawContents: yaml.dump(definitionFile),
-                    contents: definitionFile
-                })),
-                [RelativeFilePath.of(FERN_PACKAGE_MARKER_FILENAME)]: {
-                    // these files doesn't live on disk, so there's no absolute filepath
-                    absoluteFilepath: AbsoluteFilePath.of("/DUMMY_PATH"),
-                    rawContents: yaml.dump(definition.packageMarkerFile),
-                    contents: definition.packageMarkerFile
-                }
-            },
-            packageMarkers: {},
-            importedDefinitions: {}
+        getDefinition: async (language?: generatorsYml.GenerationLanguage) => {
```

Comment:
any thoughts on how to solve logging (i.e. we are going to call `loadDependency` multiple times now)

Reply:
just what we talked about already which is creating a silent logger to pass through